WcaOnRails/app/controllers/api/v0/roles_controller.rb

```diff
@@ -29,10 +29,10 @@ def show
       render json: {
         roleData: {
           delegateStatus: user.delegate_status,
-          seniorDelegateId: user.senior_delegate.id,
+          regionId: user.region_id,
           location: user.location,
         },
-        seniorDelegates: senior_delegates,
+        regions: UserGroup.regions,
       }
     else
       render json: {
```

Comment:
Should this also be replaced by `regions`?

Reply:
Oh my bad, thanks for this. This was indeed a critical bug. Sorry for missing this out.